src/aiida/storage/psql_dos/orm/authinfos.py

```
@@ -132,8 +151,23 @@ def delete(self, pk):
         session = self.backend.get_session()
 
         try:
-            row = session.query(self.ENTITY_CLASS.MODEL_CLASS).filter_by(id=pk).one()
+            row: DbAuthInfo = session.query(self.ENTITY_CLASS.MODEL_CLASS).filter_by(id=pk).one()
+            dbcomputer_pk = row.dbcomputer_id
+            dbcomputer_uuid = row.dbcomputer.uuid
+            delete_secure_password = SecureStorage.contains_redacted_password(row.auth_params)
             session.delete(row)
             session.commit()
         except NoResultFound:
             raise exceptions.NotExistent(f'AuthInfo<{pk}> does not exist')
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)